romana-install/roles/romana/install-agent/templates/config/romana.yaml

```diff
@@ -0,0 +1,16 @@
+{
+    #
+    # Default romana configuration file.
+    # please move it to ~/.romana.yaml
+    #
+    "RootURL": "{{ romana_master_ip }}",
+    "RootPort": "9600",
+    "LogFile": "/var/tmp/romana.log",
+    "Format": "table", # options are table/json 
+    {% if stack_type == "kubernetes" %}
+    "Platform": "kubernetes",
+    {% else %}
+    "Platform": "openstack",
+    {% endif %}
```

Comment:
I wonder if this should be `else if stack_type in [ "devstack", "openstack" ]` and have an empty or literal `"undefined"` value when it's none of the three.